### PR TITLE
Update france.json

### PR DESCRIPTION
**Pull request by Praxthisnovcht**

### CODE REVIEW

File: PokemonGo.RocketAPI.Console/Lang/france.json [hunk 1]
```diff
@@ -27,16 +27,18 @@
...
+    "germanPokemonNames": "Noms allemands pour les Pokémons",
     "useIncese": "Utiliser un encens toutes les 30min",
-    "evolvePokemonIfEnoughCandy": "Evoluer les Pok�mons si assez de bonbons",
-    "enablePokemonListGUI": "Activer Liste GUI Pok�mon",
-    "keepPokemonWhichCanBeEvolved": "Conserver les Pok�mons pouvant �voluer"
-}
\ No newline at end of file
+    "evolvePokemonIfEnoughCandy": "Evoluer les Pokémons si assez de bonbons",
+    "enablePokemonListGUI": "Activer Liste GUI Pokémon",
+    "keepPokemonWhichCanBeEvolved": "Conserver les Pokémons pouvant évoluer"
```
Comment (jaymoulin) [inline PokemonGo.RocketAPI.Console/Lang/france.json]: Missing comma at the end of line 41

**Comment by Gunjha** [comment]
We need to make a version without any accent.
Want me to do it ?
"é" and "è" and "à" doesn't show properly

File: PokemonGo.RocketAPI.Console/Lang/france.json (hunk 1 above)
Comment (Gunjha) [inline PokemonGo.RocketAPI.Console/Lang/france.json]: +    "pokemonNotToEvolve": "Pokemons - A evoluer",

**Comment by Praxthisnovcht** [comment]
I'll do

**Comment by BlackHatching** [comment]
Heureusement j'lai dit toute à l'heure sinon jamais ils le feraient...

**Comment by Lexobama** [comment]
All names of Pokemon are in English.

**Comment by k756u** [comment]
j'ai traduit la liste des pokemon francais english et german si besoin

**Comment by Praxthisnovcht** [comment]
normal, there's no translation for Pokemon for now
it translates the application only

**Comment by k756u** [comment]
d'accord bien s'il ya besoin pour plus tard je suis la :)

**Comment by Targolo** [comment]
Please, do not use this french translation file. I've fixed the accentuation problem in another pull request. The accentuation will now appear correctly.

**Pull request closed without merging**